### DISCUSSION TITLE
Captive Portal

### DISCUSSION THREAD

**Discussion by Fernando4242** [gjroots/esp32_nat_router_plus#26] [Q&A]
Would it be possible to implement a captive portal through the UI interface at some point or me modifying the code? If so, what would be the best way to approach for that?

**Comment by gjroots** [comment]
Hi, @Fernando4242 .

Yes everything is possible, but you need to understand the concept of project and captive portel. 

It's easy and not much difficult. It depends on your purpose.

**Reply by gjroots** [reply]
If you are willing, I guide you.
